js/RateCurve.js

```diff
@@ -75,8 +79,7 @@ RateCurve.prototype.rcCommandRawToDegreesPerSecond = function (rcData, rate, rcR
             angleRate = (rate + 27) * inputValue / 16;
```

Comment:
It is very easy to have realistic rates actually. The formula is now even more clear